src/Tabs/style.css

```diff
@@ -1,6 +1,6 @@
 .ola_tabs {
     display: flex;
-    height: var(--size-8);
+    height: calc(var(--size-8)*1.4);
```

Comment:
Using random values is agains the style guide. Why do you want to increase the height?

Reply:
in the design it has a higher height, --size-8 is small and --size-9 is too high. How can I do it? @oscarotero

Reply:
It's not important, we can left in --size-8 and I can add a padding top to tabs. forget it

Reply:
Ok, understood.
I think the problem is you've used a ola_button for that.
Maybe something like `.ola_menu-option` is better (https://github.com/marketgoo/Ola/blob/development/src/Menu/style.css#L6) (look like it doesn't have a fixed width but a padding).

You should create a `ola_tabs-option` and apply some similar styles.

Reply:
Yeah, it's better option, I changed it, please, check when you can @oscarotero